hi-ml-azure/src/health_azure/utils.py

```diff
@@ -547,6 +547,30 @@ def create_from_matching_params(from_object: param.Parameterized, cls_: Type[T])
     return c
 
 
+def create_v2_job_command_line_args_from_params(script_params: List[str]) -> str:
+
+    command_line_arg_str = ""
+    for script_param in script_params:
+        next_param_str: str
+        if "'" in script_param and '"' in script_param:
+            raise ValueError(
+                f"Script parameters cannot contain both single and double quotes. Problematic parameter: {script_param}"
+            )
+        elif "'" in script_param:
+            next_param_str = f'"{script_param}"'
+        elif '"' in script_param:
+            next_param_str = f"'{script_param}'"
+        else:
+            next_param_str = f'{script_param}'
+
+        if command_line_arg_str == "":
+            command_line_arg_str = next_param_str
+        else:
+            command_line_arg_str = f"{command_line_arg_str} {next_param_str}"
```

Comment:
It would be much easier to understand the function if you used a list comprehension to convert individual parameters and then join with " ".